wolox-books/src/app/screens/BooksList/components/shopping-list/shopping-list.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="modal-window">
+  <div class="shopping-list">
+    <div class="shopping-header">
+      <h1 class="shopping-title">Tu carrito de compras</h1>
+      <a class="close-icon" (click)="closeModal.emit();"></a>
+    </div>
+    <ul class="books-list">
+      <li *ngFor="let book of books | async; let i = index" class="book-item">
+        <span class="book-description">{{ book.title }} - {{ book.author }}</span>
+        <a class="delete-icon" (click)="removeBook(i)"></a>
```

Comment:
Should be `button`. Also, I think the book has an id, use that instead of the index

Reply:
But the ID is different than the index, the index represents the position of the book in the array of Book[] Observable object (the store), and the ID is the id of the object coming from the backend, if I used the ID, I would had to make the reducer action more complex because it would had to search of the position of the array that has the book with that particular ID... I don't know if that's clearer, but if you want to, I make the change anyways